Part2_Map_Filter_Sorted/filter.py

```diff
@@ -0,0 +1,30 @@
+#Filter
+#1
+def pos(x):
+    return x>0
+
+def pos_filter(lst):
+    ans = filter(pos, lst)
+    return list(ans)
+print(pos_filter([4, -3, 2]))
+
+#2
+def check_cap(s):
+    return (s[0] == s[0].upper())
+
+def upper_filter(lst):
+    ans = filter(check_cap, lst)
+    return list(ans)
+print(upper_filter(["We", "are", "Novobi"]))
+
+#3
+dict = {"b": 4, "c": 5}
+def check_exist(dict):
+    if (dict.get("a") == None):

```

Comment:
Ở đây anh không khuyến khích dùng get, vì nếu trong `dict` không có `key = "a"`, nó sẽ tạo ra key "a", nên dùng `"a" not in dict.keys()`